.github/workflows/tests.yml

```diff
@@ -5,6 +5,7 @@ on:
   push:
     branches:
       - main
+      - 'check/**'
```

Comment:
At first, getting CI tests to run prior to first merge here in this upstream repo was a little tricky (github blocks first time contributors from workflows).  So, it was useful to have a branch to push to in the fork for a lint/test.

Otherwise, there are some surprises for contributors when they send in a PR only to get lint-rejected days later.

This can be removed/reconfigured in many ways to achieve the same goal.  What do you think?

(meant to remove this and make it a separate issue, but since we're both looking at it, might as well ask :) )

Reply:
Thanks for the  explanation and we'll note the less than ideal user experience as a first time contributor. 

I think we can remove it.

Reply:
The relevant commit has been removed